docs/eventing-api.md

```diff
@@ -76,7 +76,7 @@ Channelable ObjectReferences and access their subscription and address data.  Th
 <td>
 <code>metadata</code><br/>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta">
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#objectmeta-v1-meta">
```

Comment:
Aren't we using the v1.21 (and ready for the 1.22)?

Reply:
Thank you @gabo1208 . I will update it with v1.21.